sdcard/os/archlinux.sh

```diff
@@ -52,7 +54,10 @@ checkrootfs(){
 
 # Format the SD Card with two partitions, boot and root. Boot is $1 MB big.
 # This makes the root partition as big as the sd card minus boot
-generalformat(){
+twopartitionformat(){
+    if [[ ! -z $SDCARDSIZE ]]; then
```

Comment:
actually, I removed these as they're not longer required, so you may safely remove them